cmd/crc/cmd/delete.go

```diff
@@ -2,14 +2,17 @@ package cmd
 
 import (
 	"github.com/spf13/cobra"
+	"os"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/input"
 	"github.com/code-ready/crc/pkg/crc/output"
 
 	"github.com/code-ready/crc/pkg/crc/machine"
 )
 
 func init() {
+	deleteCmd.Flags().BoolVarP(&clearCache, "clear-cache", "", false, "Clear the cache directory: $HOME/.crc")
```

Comment:
As per message it should be `$HOME/.crc/cache` or `fmt.Sprintf("Clear the cache directory: %s", constants.MachineCacheDir)`